data-otservbr-global/lib/core/storages.lua

```diff
@@ -81,13 +79,11 @@ Storage = {
 	-- General storages
 	Dragonfetish = 30003,
 	EdronRopeQuest = 30004,
-	GhostShipQuest = 30005,
```

Comment:
```suggestion
        -- GhostShipQuest = 30005, (moved to kv)
```

Reply:
why? i didnt get it

Reply:
mantain removed storages as a comment. to save an "history" for the future, don't use the same id.

Reply:
even with the migration file?